recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "5.1.0" %}
+{% set version = "5.1.1" %}
 
 package:
   name: qtconsole
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/q/qtconsole/qtconsole-{{ version }}.tar.gz
-  sha256: 12c734494901658787339dea9bbd82f3dc0d5e394071377a1c77b4a0954d7d8b
+  sha256: bbc34bca14f65535afcb401bc74b752bac955e5313001ba640383f7e5857dc49
 
 build:
   noarch: python
```

Comment:
Please add s390x to the skipped section.  s390x has as well as ppc64le no qt support, therefore it should be skipped

Reply:
I've skipped s390x as well as ppc64le, if we only want to skip s390x I will change it.